content/docs/command-reference/stage/add.md

```diff
@@ -0,0 +1,432 @@
+
+- [external dependencies](/doc/user-guide/external-dependencies) and
+  [external outputs](/doc/user-guide/managing-external-data) (outside of the
+  <abbr>workspace</abbr>) are also supported (except metrics and plots).
+
+- Outputs are deleted from the workspace before executing the command (including
+  at `dvc repro`) if their paths are found as existing files/directories (unless
+  `--outs-persist` is used). This also means that the stage command needs to
+  recreate any directory structures defined as outputs every time its executed
+  by DVC.
+
+- In some situations, we have previously executed a stage, and later notice that
+  some of the files/directories used by the stage as dependencies, or created as
+  outputs are missing from `dvc.yaml`. It is possible to
+  [add missing dependencies/outputs to an existing stage](/docs/user-guide/how-to/add-deps-or-outs-to-a-stage)
+  without having to execute it again.
+
+- Renaming dependencies or outputs requires a
+  [manual process](/doc/command-reference/move#renaming-stage-outputs) to update
+  `dvc.yaml` and the project's cache accordingly.
+
+### For displaying and comparing data science experiments
+
+[parameters](/doc/command-reference/params) (`-p`/`--params` option) are a
+special type of key/value dependencies. Multiple parameter dependencies can be
+specified from within one or more YAML, JSON, TOML, or Python parameters files
+(e.g. `params.yaml`). This allows tracking experimental hyperparameters easily.
+
+Special types of output files, [metrics](/doc/command-reference/metrics) (`-m`
+and `-M` options) and [plots](/doc/command-reference/plots) (`--plots` and
+`--plots-no-cache` options), are also supported. Metrics and plots files have
+specific formats (JSON, YAML, CSV, or TSV) and allow displaying and comparing
+data science experiments.
+
+### The `command` argument
+
+The `command` sent to `dvc stage add` can be anything your terminal would accept
+and run directly, for example a shell built-in, expression, or binary found in
+`PATH`. Please remember that any flags sent after the `command` are considered
+part of the command itself, not of `dvc stage add`.
+
+⚠️ While DVC is platform-agnostic, the commands defined in your
+[pipeline](/doc/command-reference/dag) stages may only work on some operating
+systems and require certain software packages to be installed.
+
+Wrap the command with double quotes `"` if there are special characters in it
+like `|` (pipe) or `<`, `>` (redirection), otherwise they would apply to
+`dvc stage add` itself. Use single quotes `'` instead if there are environment
+variables in it that should be evaluated dynamically. Examples:
+
+```dvc
+$ dvc stage add -n first_stage "./a_script.sh > /dev/null 2>&1"
+$ dvc stage add -n second_stage './another_script.sh $MYENVVAR'
+```
+
+## Options
+
+- `-n <stage>`, `--name <stage>` (**required**) - specify a name for the stage
+  generated by this command (e.g. `-n train`). Stage names can only contain
```

Comment:
Quick Q: Are there any changes to options text vs. https://dvc.org/doc/command-reference/run#options ?

Reply:
No changes on options text, just few options that were related to running  a stage are not in stage:add which were removed.